### DISCUSSION TITLE
Ngrok

### DISCUSSION THREAD

**Discussion by yesyeahvh123** [gutris1/segsmaker#26] [Q&A] [answered]
i cant connect with ngrok, even though I tried many times with valid token but ngrok keep say token not valid, can you check it, and can you add cloudflare tunned? thanks

**Comment by gutris1** [comment]
> i cant connect with ngrok, even though I tried many times with valid token but ngrok keep say token not valid, can you check it, and can you add cloudflare tunned? thanks

it's your token and it's your ngrok account, how am I supposed to check?
also cloudflared doesnt work on studio lab.

**Comment by yesyeahvh123** [comment]
i check token and it work with google colab so i dont think the problem is token

**Reply by gutris1** [reply]
alright, I'll check them later

**Comment by gutris1** [comment, marked as answer]
rerun this cell, FORGE or AUTO1111
![image](https://github.com/gutris1/segsmaker/assets/132797949/f30b523e-727d-46b3-abd7-1441e8be8914)
<br />
<br />
<br />

then rename ngrok.py to ngrokk.py
<br />
![image](https://github.com/gutris1/segsmaker/assets/132797949/e6d38de0-ea9a-421a-9618-42052d8db1eb)

**Comment by yesyeahvh123** [comment]
still error 
launch.py: error: argument --ngrok: expected one argument
ngrok authtoken detected, trying to connect...
Invalid ngrok authtoken? ngrok connection aborted due to: module 'ngrok' has no attribute 'connect'
does i need clean all and clear install?

**Reply by gutris1** [reply]
did you read my post above, I post two image

**Reply by yesyeahvh123** [reply]
yes already do it

**Reply by gutris1** [reply]
it work on me 
![image](https://github.com/gutris1/segsmaker/assets/132797949/4d74898f-cdf1-4cf6-a989-2f1495d6b5ac)

**Reply by yesyeahvh123** [reply]
i dont known why, i will try fresh install

**Reply by yesyeahvh123** [reply]
@gutris1 after fresh install it work now thanks

**Reply by gutris1** [reply]
yeah sure

**Reply by yesyeahvh123** [reply]
@gutris1 does sagemaker support https://localhost.run and https://hrzn.run ? thanks

**Reply by gutris1** [reply]
no

**Comment by mQuan88** [comment]
Me too, but with ComfyUI, tried to run web ui but the error message come out
_sh: 1: /tmp/venv/bin/python3: not found_
don't know why and how to fix it, please help!

**Reply by gutris1** [reply]
see changelog
